src/main/java/school/lemon/changerequest/java/introduction/hw1/Task10.java

```diff
@@ -4,6 +4,11 @@
 public class Task10 {
 
     public static boolean containDigitTwo(int n) {
+
+        n = Math.abs(n);
```

Comment:
Why do you need `abs` here?

Reply:
If n < 0, n % 10 == -2, so by using abs we do not need to add one more condition to if. I am not sure which solution is better.